### ISSUE TITLE
Testing of neural network

### ISSUE THREAD

**Issue by roddar92** [Svetuf/CSC_Sentiment_Russian_NER#7]
Input layer (words' indices and indices of smiles)
 ||
Embedding (dim >= 32)
||
Convolution layer 1 (filters <= embeddings_dim, kernel_size 4|8)
||
Convolution layer 2 (filters <= embeddings_dim, kernel_size 4|8)
||
Convolution layer 3 (filters <= embeddings_dim, kernel_size 4|8)
||
Bidirectional LSTM (units = 0.5 * LSTM_units), LSTM_units <= embeddings_dim
||
Dense (activation='softmax')

It is a part of a Tacotron (encoder)
